web/cobrands/sass/_base.scss

```diff
@@ -1957,6 +1957,85 @@ html.js #map .noscript {
   @include svg-background-image("/cobrands/fixmystreet/images/map-tools");
 }
 
+// Map instructions for keyboard
+.keyboard-intructions-wrapper {
+  display:flex;
+  position: absolute;
+  padding: 10px 25px;
+  padding-top: 14px;
+  background-color: white;
+  border-radius: 8px;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  bottom: 35px;
+  left: 50%;
+  transform: translateX(-50%);
+
+  span {
+    display: inline-block;
+    margin-left: 20px;
+    font-weight: 600;
+  }
+
+  &.basic {
+    opacity: 0;
+  }
+
+  @media only screen and (max-width:48em) {
+    display: none;
+  }
+}
+
+.keyboard-key-icon {
+  display: inline-block;
+  border-radius: 4px;
+  width: 30px;
+  height: 0;
+  padding-top: 22px;
+  overflow: hidden;
+  background-color: #ECEBEB;
+  box-shadow: 0px 3px 0px rgba(0, 0, 0, 0.3);
+  background-repeat: no-repeat;
+  background-position: center;
+
+  &.zoom-in {
+    background-image: url("data:image/svg+xml,%3Csvg fill='none' height='14' viewBox='0 0 13 14' width='13' xmlns='http://www.w3.org/2000/svg'%3E%3Cg fill='%23222'%3E%3Crect height='13' rx='.3' width='3.25' x='4.875' y='.5'/%3E%3Crect height='13' rx='.3' transform='matrix(0 -1 1 0 -8.625 8.625)' width='3.25' y='8.625'/%3E%3C/g%3E%3C/svg%3E");
+    background-size: 13px;
+  }
+  &.zoom-out {
+    background-image: url("data:image/svg+xml,%3Csvg fill='none' height='4' viewBox='0 0 15 4' width='15' xmlns='http://www.w3.org/2000/svg'%3E%3Crect fill='%23222' height='14' rx='.3' transform='matrix(0 -1 1 0 -3.25 4.25)' width='3.5' x='.5' y='3.75'/%3E%3C/svg%3E");
+    background-size: auto 4px;
```

Comment:
Assuming we keep these as background images, it would be _really nice_ if all of the SVGs had the same dimensions (say, 16x16px) so that we didn’t have to mess with different background-sizes for each one. Obviously, the shapes inside the SVG could be wider or narrower, etc, to maintain a consistent thickness / visual weight across the set, but I think an identical "canvas" size for each one, would make everything simpler.